vignettes/plotting_vignette.Rmd

```diff
@@ -0,0 +1,160 @@
+---
+title: "Plotting Functionality in tcplfit2"
+author: "Center for Computational Toxicology and Exposure"
+output:
+   prettydoc::html_pretty:
+    theme: architect
+    toc: yes
+    toc_depth: 4
+vignette: >
+  %\VignetteIndexEntry{Plot_with_tcplfit2}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, packages, warning=FALSE, message=FALSE}
+library(tcplfit2)
+library(dplyr)
+library(stringr)
+library(ggplot2)
+library(gridExtra)
+```
+
+```{r, include=FALSE}
+source("~/CompTox-ToxCast-tcplFit2/R/concRespPlot2.R")
+source("~/CompTox-ToxCast-tcplFit2/R/plot_allcurves.R")
+```
+
+
+# Introduction
+This vignette demonstrates how to plot concentration-response curves with two `ggplot2` based plotting functions in `tcplfit2`, namely `plot_allcurves` and `concRespPlot2`. 
+
+`plot_allcurves` plots the observed concentration-response data along with all the model fits, and `concRespPlot2` plots a basic plot of the data with the winning model. Both functions return `ggplot` objects which are capable of being customized to have additional details, such as labels for cutoff, BMR, BMD, BMD confident interval, and other curve fits, etc. with `ggplot2` add-ons. How to add layers to the plot will also be demonstrated in the vignette. 
```

Comment:
Please consolidate these descriptions in the descriptions that are part of the respective sections.